src/planner/vectorfield/VectorFieldPlanner.cpp

```diff
@@ -141,6 +141,14 @@ std::unique_ptr<aikido::trajectory::Spline> planToEndEffectorOffset(
     std::chrono::duration<double> timelimit,
     planner::PlanningResult* planningResult)
 {
+  // ensure that no two planners run at the same time
+  if (metaskeleton->getNumBodyNodes() <= 0)
```

Comment:
Nit: This is technically fine, but negativity check is unnecessary here because the return type is unsigned.